turbo.json

```diff
@@ -29,7 +29,6 @@
     "lint": {
       "outputs": []
     },
-    "lint-fix": {
-    }
+    "lint-fix": {}
```

Comment:
An example of a misconfigured code formatting. Something we will fix with this pull request forever.